gauges/backends.go

```diff
@@ -31,6 +31,18 @@ func (g *Gauges) MaxBackends() prometheus.Gauge {
 	)
 }
 
+// CurrentBackends returns the number of backends currently connected to all databases 
+func (g *Gauges) CurrentBackends() prometheus.Gauge {
+	return g.new(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_current_backends",
```

Comment:
```suggestion
			Name:        "postgresql_instance_connected_backends",
```
maybe sounds better for the purpose of this metric, if you accept the change is a good idea rename the func name to `InstanceConnectedBackends` too.

Reply:
Sounds good to me. I'll be changing it.